hipcheck/src/plugin/supported_arch.rs

```diff
@@ -0,0 +1,44 @@
+use std::{fmt::Display, str::FromStr};
+
+use crate::hc_error;
+
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
+/// Officially supported target triples, as of RFD #0004
+///
+/// NOTE: these architectures correspond to the offically supported Rust platforms
+pub enum SupportedArchitecture {
```

Comment:
My only thought is this could probably be shortened to `SupportedArch`. Otherwise LGTM!

Reply:
Updated it!